test/ItspServices.pServer.Test/AuthorizedEndpointTest.cs

```diff
@@ -213,50 +164,81 @@ public async Task ProtectedDataGetFolderById()
         [TestMethod]
         public async Task ProtectedDataGetById()
         {
-            ProtectedDataRepository.Setup(x => x.GetById(0)).Returns(Data0);
+            ProtectedData data = new ProtectedData()
+            {
+                Name = "Data",
+                Data = Encoding.UTF8.GetBytes("DATA")
+            };
+            data.Users.RegisterEntries.Add(new UserRegisterEntry()
+            {
+                User = User,
+                Permission = Permission.READ,
+                EncryptedKeys = new[]
+                {
+                    new SymmetricKey() { KeyData = Encoding.UTF8.GetBytes("SymmetricKey1"), MatchingPublicKeyId = 0 },
+                    new SymmetricKey() { KeyData = Encoding.UTF8.GetBytes("SymmetricKey2"), MatchingPublicKeyId = 1 }
+                }.ToList()
+            });
+
+            ProtectedDataRepository.Setup(x => x.GetById(0)).Returns(data);
 
             var response = await UserClient.GetAsync("/api/protecteddata/data/0");
             Assert.AreEqual(HttpStatusCode.OK, response.StatusCode);
             string content = await response.Content.ReadAsStringAsync();
-            DataModel data = JsonConvert.DeserializeObject<DataModel>(content);
-
-            Assert.AreEqual(Data0.Name, data.Name);
-            CollectionAssert.AreEquivalent(Data0.Data, data.Data);
-            CollectionAssert.AreEquivalent(User.PublicKeys[0].KeyData, data.KeyPairs.ToList()[0].PublicKey);
-            CollectionAssert.AreEquivalent(Data0.Users.RegisterEntries[0].EncryptedKeys[0].KeyData, data.KeyPairs.ToList()[0].SymmetricKey);
-            CollectionAssert.AreEquivalent(Data0.Users.RegisterEntries[0].EncryptedKeys[1].KeyData, data.KeyPairs.ToList()[1].SymmetricKey);
+            DataModel dataModel = JsonConvert.DeserializeObject<DataModel>(content);
+
+            Assert.AreEqual(data.Name, dataModel.Name);
+            CollectionAssert.AreEquivalent(data.Data, dataModel.Data);
+            CollectionAssert.AreEquivalent(User.PublicKeys[0].KeyData, dataModel.KeyPairs.ToList()[0].PublicKey);
+            CollectionAssert.AreEquivalent(User.PublicKeys[1].KeyData, dataModel.KeyPairs.ToList()[1].PublicKey);
+            CollectionAssert.AreEquivalent(data.Users.RegisterEntries[0].EncryptedKeys[0].KeyData, dataModel.KeyPairs.ToList()[0].SymmetricKey);
+            CollectionAssert.AreEquivalent(data.Users.RegisterEntries[0].EncryptedKeys[1].KeyData, dataModel.KeyPairs.ToList()[1].SymmetricKey);
         }
 
         [TestMethod]
         public async Task UnauthorizedRequestData()
         {
-            ProtectedDataRepository.Setup(x => x.GetById(0)).Returns(DataWithoutRegisteredUsers);
-            var response = await UserClient.GetAsync("/api/protecteddata/data/0");
+            ProtectedData data = new ProtectedData()
+            {
+                OwnerId = 999
```

Comment:
Would change the OwnerId here to 1234 to prevent to get confused with the data id.